src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function (compareFunction = defaultCompareFunction) {
```
move defaultCompareFunction outside of sort2 function and use it as default argument.
in method work with compareFunction (no need make compare variable)